hackathon_site/dashboard/frontend/src/pages/Inventory/Inventory.tsx

```diff
@@ -107,7 +114,12 @@ const Inventory = () => {
                                 </Hidden>
 
                                 <div className={styles.inventoryBodyToolbarRefresh}>
-                                    <Typography variant="body2">123 items</Typography>
+                                    <Typography
+                                        variant="body2"
+                                        data-testid="inventory-total-count"
+                                    >
+                                        {count} items
```

Comment:
Hmm, does `{count} items` make sense? With that terminoligy, I would expect that to be the number of items displayed, not total.

The figma uses `{count} total results`, could we switch this to that or `{count} results`? (I prefer the shorter one, I defer to you for which one to pick).

Reply:
The shorter the better, ppl don't like to read 👍